### PR TITLE
Collab work

### PR DESCRIPTION
**Pull request by bserge13**
Me and Justin really did a lot...not sure what we did, but we did a lot of it. 

Removal of outdated django requirement (distutils) that is no longer applicable in update python versions (post-python 3.9). Substituted with require environ plug ins, snippets, and syntaxes. Migrations not completed, need to pull Brendans version of a created super user down to be able to sync the local requirements for setting local server....but besides that, we're great :')

### CODE REVIEW

File: backend_api/models.py [hunk 1]
```diff
@@ -7,15 +7,15 @@ class User(models.Model):
...
+
 class Reviews(models.Model): 
   CATEGORY_TAGS = [
-    ('Security', 'Security'),
-    ('Restaurants', 'Restaurants'),
-    ('Bathrooms', 'Bathrooms'),
-    ('General', 'General'),
-    ('Amenities', 'Amenities'),
-    ('Accessibility', 'Accessibility')
+    ('security', 'Security'),
```
Comment (JustinSteel) [inline backend_api/models.py]: is this a formatting thing?

File: backend_api/models.py [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+#   airport = models.ForeignKey() #(Airport, on_delete=models.CASCADE) 
+#   comment = models.TextField(max_length=150)
+#   category = models.CharField(tags=CATEGORY_TAGS)
 
-  user = models.ForeignKey(User, on_delete=models.CASCADE)
-  airport = models.ForeignKey(Airport, on_delete=models.CASCADE) 
-  comment = models.TextField(max_length=150)
-  category = models.CharField(tags=CATEGORY_TAGS)
\ No newline at end of file
+#   def __str__(self):
```
Comment (JustinSteel) [inline backend_api/models.py:27]: we still have to uncomment these

File: runway_reviews_be/settings.py [hunk 3]
```diff
@@ -9,18 +9,22 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/5.0/ref/settings/
 """
+import environ
```
Comment (JustinSteel) [inline runway_reviews_be/settings.py:12]: nice workaround

**Review by JustinSteel — APPROVED** [review]

**Comment by brendan-bondurant** [comment]
Cool, this updated file structure will work well. I encountered a few obstacles related to that when creating the new user. I'll merge this and pull it into the branch to see how it lines up. I won't do any more work until I've talked to you and ensured we're on the same page with the structure.

**Pull request merged**